operator/tests/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Runs operator e2e tests.
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+
+source "$ROOT/scripts/lib.sh"
+source "$ROOT/scripts/ci/lib.sh"
+
+test_operator_e2e() {
+    info "Starting oerator e2e tests"
+
+    require_environment "KUBECONFIG"
+
+    read -r -d '' kuttl_help <<- _EO_KUTTL_HELP_ || true
+See log and/or kuttl JUnit output for error details.
+Reading operator/tests/TROUBLESHOOTING_E2E_TESTS.md may also be helpful.
+_EO_KUTTL_HELP_
+
+    rm -f FAIL
```

Comment:
I'm not a big fan of littering $PWD with files like this. I think all of the commands below could just use an in-memory variable, see example below...

Reply:
Yeah. This is a bad habit carried over from Circle CI steps. Will variabilize it.